enterprise/views.py

```diff
@@ -582,7 +583,8 @@ def _enroll_learner_in_course(
         if course_id and self.is_course_run_id(course_id):
             if license_uuid:
                 enrollment_api_client = EnrollmentApiClient()
-                existing_enrollment = enrollment_api_client.get_course_enrollment(request.user.username, course_id)
+                existing_enrollment = enrollment_api_client.get_course_enrollment(request.user.username,
+                                                    course_id)
```

Comment:
[style nitpick] If the args make this line too long, write it like
```python
existing_enrollment = enrollment_api_client.get_course_enrollment(
    request.user.username, course_id
)
```

Reply:
@iloveagent57  It's done